jemalloc-sys/build.rs

```diff
@@ -39,8 +49,23 @@ fn main() {
     let cflags = compiler.args().iter().map(|s| s.to_str().unwrap())
                          .collect::<Vec<_>>().join(" ");
 
+
+    let configure = src_dir.join("jemalloc/configure");
+    if !configure.exists() {
+        // maybe forgot to update git submodules?
+        let mut git_cmd = Command::new("git");
+        git_cmd.arg("submodule")
+            .arg("update")
+            .arg("--init")
+            .arg("--recursive");
+        run (&mut git_cmd);
+    }
+    if !configure.exists() {
+        panic!("jemalloc/configure does not exist");
+    }
```

Comment:
Can this be left out to a future PR?

The submodule update here I don't think shoudl be run with `run` which asserts success, it should just be opportunistic and should provide a better failure message unrelated to git if the git step ends up failing.

Reply:
@alexcrichton the submodule update is a mistake in this PR, i have reverted it. 

I have it in a separate PR, but `git add -u` pulled into all my other PRs making `i686-apple` builds fail... Sorry about that.